Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -356,6 +358,16 @@ defineSuite([
         });
     });
 
+    fit('loads tileset with extras', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetWithExtras).then(function(tileset) {
+            expect(tileset.extras).toBeDefined();
+            expect(tileset.extras).toEqual({'key': 'value'});
+
+            expect(tileset.root.extras).toBeDefined();
+            expect(tileset.root.extras).toEqual({'key': 'value'});
```

Comment:
Since you mentioned it in the PR description, it might be worthwhile to add a test that checks that extras is undefined for some other tile in the tree that doesn't have extras.